covidanalyzer.py

```diff
@@ -0,0 +1,120 @@
+import csv
+
+
+def total_country_cases_deaths(total_country,
+                               total_cases,
+                               total_deaths,
+                               total_recovered_cases):
+    with open('covid_cases_stats.csv') as file_obj:
+        reader_obj = csv.DictReader(file_obj)
+        for row in reader_obj:
+            if row['country'] != 'Total:':
+                if row['country']:
+                    total_country += 1
+                if row['total_deaths']:
+                    total_deaths += int(row['total_deaths'])
+                if row['total_cases']:
+                    total_cases += int(row['total_cases'])
+                if row['total_recovered']:
+                    total_recovered_cases += int(row['total_recovered'])
+            else:
+                pass
```

Comment:
If we don't have any condition to add in else then we don't have to add it
Remove else and pass

Reply:
Resolved..